nodestream_akamai/netstorage/netstorage.yaml

```diff
@@ -0,0 +1,81 @@
+- implementation: nodestream-akamai.netstorage:AkamaiNetstorageExtractor
+  arguments:
+    base_url: !env 'base_url'
+    client_token: !env 'client_token'
+    client_secret: !env 'client_secret'
+    access_token: !env 'access_token'
+    account_key: !env 'account_key'
+  annotations:
+  - live
+  - test
+- implementation: nodestream.interpreting:Interpreter
+  arguments:
+    interpretations:
+    - type: source
+      node_type: akamai_netstorage_config
+      normalization:
+        do_remove_trailing_dots: true
+      key:
+        storageGroupId: !jmespath 'storageGroupId'
+    - type: properties
+      properties:
+        storageGroupName: !jmespath 'storageGroupName'
+        estimatedUsageGB: !jmespath 'estimatedUsageGB'
+        domainPrefix: !jmespath 'domainPrefix'
+    - type: relationship
+      node_type: akamai_property
+      relationship_type: HOSTS_CONTENT_FOR
+      key_normalization:
+        do_remove_trailing_dots: true
+      node_key:
+        origins[*].name: !jmespath 'downloadDomain'
```

Comment:
This seems a little messed up. The value on the left should be a name of the key for the node, this seems to be a jmespath. I don't think the script messed this up, because the rest seem correct. But its worth confirming.

Reply:
Yeah i'm not sure about this. The rewriter script also had the wrong implementation value of `nodestream.interpreters:Interpreter` which recently changed to `nodestream.interpreting:Interpreter` so some of it could definitely be wrong. I had to update the interpreting manually. but all the properties in side should have stayed the same I believe.

Reply:
Heres the original: https://github.com/nodestream-proj/neo4j-akamai/blob/main/neo4j_netstorage_loader/akamai-netstorage.yaml#L25

Reply:
This does appear to be a typo in the original script:

https://github.com/nodestream-proj/neo4j-akamai/blob/main/neo4j_netstorage_loader/akamai-netstorage.yaml#L25